pkgs/top-level/python-packages.nix

```diff
@@ -1177,6 +1177,8 @@ in {
 
   basemap = callPackage ../development/python-modules/basemap { };
 
+  baserow = callPackage ../servers/baserow { };
```

Comment:
This should be outside of pythonPackages which gives us the option to overwrite inputs